docs/ota-client-guide/modules/ROOT/pages/comparing-full-filesystem-update-strategies.adoc

```diff
@@ -0,0 +1,29 @@
+= Comparing full-filesystem update strategies
```

Comment:
This is missing the `We recommend that you view this article in our documentation portal` header.

Reply:
Fixed